config/webpack.common.config.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: [
+    path.resolve(__dirname, '../src/index.jsx'),
+  ],
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, '../dist'),
+  },
+  resolve: {
+    extensions: ['.js', '.jsx'],
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
```

Comment:
We may not want to use this plugin in production if we are just outputting js files for studio's html to consume.

Reply:
Ah, good point. Also, what do you think about naming the output bundle `studio-frontend.js` in prod?

Reply:
sounds good to me!